algorithms-modules/algorithms-miscellaneous-7/src/test/java/com/baeldung/algorithms/connect4/GameLiveTest.java

```diff
@@ -0,0 +1,40 @@
+package com.baeldung.algorithms.connect4;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class GameLiveTest {
```

Comment:
This could simply run as a unit test with the regular builds - no need to make it a "LiveTest".

Reply:
Well dang - it was meant to be and I apparently just used the wrong word!